frappe_pwa/frappe_pwa/doctype/web_app_manifest/web_app_manifest.py

```diff
@@ -7,11 +7,19 @@
 
 from frappe.model.document import Document
 
+
 class WebAppManifest(Document):
 
-	def on_update(self):
-		"""clear cache"""
-		frappe.clear_cache(user = 'Guest')
+    def on_update(self):
+        """clear cache"""
+        frappe.clear_cache(user='Guest')
+
+        from frappe.website.render import clear_cache
+        clear_cache()
 
-		from frappe.website.render import clear_cache
-		clear_cache()
+    def configure_pwa(self):
+        ws = frappe.get_doc('Website Settings')
+        ws.head_html = '''<link href="/assets/frappe_pwa/manifest.json" rel="manifest">'''
+        ws.save()
```

Comment:
@AminovE99 As discussed before, we should:

- Check if `rel="manifest"` is in `ws.head_html`
- If not present, **append** `<link href="/assets/frappe_pwa/manifest.json" rel="manifest">` at the end and **not erase the existing headers